package.json

```diff
@@ -29,7 +29,7 @@
     "husky": "^8.0.0",
     "minimist": "1.2.6",
     "mocha": "^10.0.0",
-    "ndb": "1.1.5",
```

Comment:
@beblicarl please why this change?

Reply:
I was told to update my npm , mongo and everything else including force audit for the error I was facing to go but it still didn't. I'm sure it's one of the things I installed

Reply:
not sure how an old version helps but cool